protocol/protocol.tex

```diff
@@ -5812,64 +7020,69 @@
 \vspace{-0.25ex}
   \canopyonwarditem{let $\NoteCommitRandBytes = \begin{cases}
                       \NoteSeedBytes,&\caseif \NotePlaintextLeadByte = \hexint{01} \\
```

Comment:
This is already covered by the fact that NU5 activates after the ZIP 212 grace period has finished.

Reply:
Actually I decided to split this into separate Sapling and Orchard sections, since there are sufficient differences to warrant that. The Orchard version does not need to consider the pre-ZIP-212 case.